sound/sox_ng/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sox_ng
+PKG_VERSION:=14.5.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeberg.org/sox_ng/sox_ng/releases/download/$(PKG_NAME)-$(PKG_VERSION)
+PKG_HASH:=1ebb6e6eedd1bb9a7b25bf54492f372d80c987f8f586d7e7cde69678889ff958
+
+PKG_LICENSE:=LGPL-2.1 GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+PKG_CPE_ID:=cpe:/a:sound_exchange_project:sound_exchange
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/sox_ng
+  SECTION:=sound
+  CATEGORY:=Sound
+  DEPENDS:=+lame-lib +libmad +libid3tag +libmagic \
+		+libvorbis +alsa-lib +libflac +libopus +libopusfile +libwavpack
+  TITLE:=Sox_ng is a general purpose sound converter/player/recorder
+  URL:=https://codeberg.org/sox_ng/sox_ng
+endef
+
+define Package/sox_ng/description
+	SoX_ng is a command line utility that can convert various formats
+	of computer audio files in to other formats. It can also apply
+	various effects to these sound files during the conversion.
+	As an added bonus, SoX_ng can play and record audio files on
+	several unix-style platforms.
+endef
```

Comment:
```suggestion
define Package/sox_ng/description
  SoX_ng is a command line utility that can convert various formats
  of computer audio files in to other formats. It can also apply
  various effects to these sound files during the conversion.
  As an added bonus, SoX_ng can play and record audio files on
  several unix-style platforms.
endef
```